metals/src/main/scala/scala/meta/internal/tvp/ClasspathTreeView.scala

```diff
@@ -102,17 +105,20 @@ class ClasspathTreeView[Value, Key](
           else MetalsCommand.goto(child.symbol)
 
         val icon = child.kind match {
-          case k.OBJECT | k.PACKAGE_OBJECT => "object"
-          case k.TRAIT => "trait"
-          case k.CLASS => "class"
-          case k.INTERFACE => "interface"
-          case k.METHOD | k.MACRO =>
-            if (child.properties.isVal) "val"
-            else if (child.properties.isVar) "var"
-            else "method"
-          case k.FIELD =>
-            if (child.properties.isEnum) "enum"
-            else "field"
+          case k.OBJECT | k.PACKAGE_OBJECT => "symbol-object"
```

Comment:
I changed these to use the VS Code defaults (which can be replaced by a theme etc.) instead of the custom one. I think it makes sense to use the same everywhere and we were missing some such as type.